src/main/java/com/kjetland/dropwizard/activemq/ActiveMQReceiverHandler.java

```diff
@@ -285,4 +300,12 @@ protected Result check() throws Exception {
         };
     }
 
+    public void addRecieverFilter(ReceiverFilter filter) {
+        receiverFilters.add(filter);
+    }
+
+    public String getDestination() {

```

Comment:
I'm not sure this will be needed anymore.
Maybe it's convenient though? If so, keep it.